### ISSUE TITLE
The FWFree record overflows because it is asynInt32

### ISSUE THREAD

**Issue by MarkRivers** [areaDetector/ADEiger#49]
The FWFree record overflows because it is asynInt32 for very large RAM disks.  It should be asynFloat64.

**Comment by MarkRivers** [comment]
This appeared to be fixed by changing the ai record to asynFloat64 and changing the eigerParam to asynFloat64.  But Kaz reported these errors:
```
2021/12/21 15:53:23.144 Param[FW_FREE]::fetch<double>: [param=FW_FREE] unexpected type 3
```
That error makes sense because the JSON type is Int.  I don't understand why those errors only happen once, and why it appears to actually work OK. 

Needs investigating.

**Comment by MarkRivers** [comment]
The errors were actually occuring whenever FW_FREE was polled.  Kaz had periodic polling disabled, so it only happened when acquisition started.

This is fixed with ed3f9de9cb758c820dabde28ed0488add9724b42.  Parameters of type int and uint in JSON can now be mapped to asynFloat64 parameters.  This eliminates the error messages.
